### PR TITLE
OKTA-452179 Tuscany nut facts cleanup for org-setup/ and run-samples/ pages

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Update nutrition facts for /oie-common-org-setup/ and /oie-embedded-common-run-samples/ pages, as well as fixing some stack selector links.
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-452179](https://oktainc.atlassian.net/browse/OKTA-452179)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/android/runsdkapp.md [hunk 1]
```diff
@@ -8,3 +8,7 @@
 1. Open the `okta.properties` file in the project root directory and [configure it](/docs/guides/oie-embedded-common-download-setup-app/android/main/#set-the-configuration-values) with the settings from your org.
 1. Click **Run** to launch the application in the Android emulator.
 1. Tap **LOGIN** to start the authentication flow.
+
+### Work with the use cases
+
+After you've successfully run the sample app, you can build your own integration by using the sample app as your guide. Explore use cases that are available with the SDK, starting with the [Basic sign-in flow using the password factor](/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/android/main/) use case.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/android/runsdkapp.md:14]:
```suggestion
After you successfully run the sample app, you can build your own integration by using the sample app as your guide. Explore use cases that are available with the SDK, starting with the [Basic sign-in flow using the password factor](/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/android/main/) use case.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/aspnet/runsdkapp.md [hunk 2]
```diff
@@ -15,3 +15,7 @@
 1. Click **Sign In**. If successful, the app redirects you to the user profile page that displays basic user profile and security token information.
 
    ![Example user profile page](/img/oie-embedded-sdk/oie-embedded-sdk-sample-app-user-profile-page.png)
+
+### Work with the use cases
+
+After you've successfully run the sample app, you can build your own integration by using the sample app as your guide. Explore use cases that are available with the SDK, starting with the [Basic sign-in flow using the password factor](/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/aspnet/main/) use case.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/aspnet/runsdkapp.md:21]:
```suggestion
After you successfully run the sample app, you can build your own integration by using the sample app as your guide. Explore use cases that are available with the SDK, starting with the [Basic sign-in flow using the password factor](/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/aspnet/main/) use case.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/aspnet/runwidgetapp.md [hunk 3]
```diff
@@ -16,8 +16,12 @@
...
 
-* If the "There was an unexpected internal error. Please try again." message is displayed instead of the Sign-In Widget, then verify that CORS is enabled. Follow the steps in [Add a trusted origin and enable CORS](/docs/guides/oie-embedded-common-org-setup/aspnet/main/#add-a-trusted-origin-and-enable-cors) to enable CORS.
+* If the "There was an unexpected internal error. Please try again." message appears instead of the Sign-In Widget, then verify that CORS is enabled. Follow the steps in [Add a trusted origin and enable CORS](/docs/guides/oie-embedded-common-org-setup/aspnet/main/#add-a-trusted-origin-and-enable-cors) to enable CORS.
+
+### Start your work with the use cases
+
+After you have successfully run the sample app, the next step is to build your integration by using the sample app as your guide. See [Load the Widget](/docs/guides/oie-embedded-widget-use-case-load/aspnet/main/) to start using the Widget and explore the available use cases.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/aspnet/runwidgetapp.md:27]:
```suggestion
After you successfully run the sample app, the next step is to build your integration by using the sample app as your guide. See [Load the Widget](/docs/guides/oie-embedded-widget-use-case-load/aspnet/main/) to start using the Widget and explore the available use cases.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/go/runsdkapp.md [hunk 4]
```diff
@@ -6,12 +6,16 @@
...
 1. Click **Sign In**.
 1. On the sign-in page, enter the username (email) and password that you used in [Create your Okta account](/docs/guides/oie-embedded-common-org-setup/go/main/#create-your-okta-account).
-1. If you've successfully signed in, the following user profile information is displayed:
+1. If you've successfully signed in, the following user profile information appears:
 
    ![The default user page that shows the user profile information](/img/oie-embedded-sdk/oie-embedded-SDK-golang-sample-app-user-default-page.png)
+
+### Work with the use cases
+
+After you've successfully run the sample app, you can build your own integration by using the sample app as your guide. Explore use cases that are available with the SDK, starting with the [Basic sign-in flow using the password factor](/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/go/main/) use case.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/go/runsdkapp.md:21]:
```suggestion
After you successfully run the sample app, you can build your own integration by using the sample app as your guide. Explore use cases that are available with the SDK, starting with the [Basic sign-in flow using the password factor](/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/go/main/) use case.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/go/runwidgetapp.md [hunk 5]
```diff
@@ -15,15 +15,18 @@ The sample app is located here in the [Golang embedded Sign-In Widget sample app
 
 1. Click **Login**.
 1. On the sign-in page, enter the username (email) and password that you've used in [Create your Okta account](/docs/guides/oie-embedded-common-org-setup/go/main/#create-your-okta-account).
-1. If you've successfully signed in, the following screen is displayed
-   with the user name:
+1. If you've successfully signed in, the following screen appears with the user name:
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/go/runwidgetapp.md:18]:
```suggestion
1. If you successfully sign in, the following screen appears with the user name:
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/go/runwidgetapp.md [hunk 6]
```diff
@@ -15,15 +15,18 @@ The sample app is located here in the [Golang embedded Sign-In Widget sample app
...
 
-## Troubleshoot
+### Troubleshoot
 
-* If the "There was an unexpected internal error. Please try again." message is displayed instead of the Sign-In Widget, then verify that CORS is enabled. Follow the steps in [Add a trusted origin and enable CORS](/docs/guides/oie-embedded-common-org-setup/go/main/#add-a-trusted-origin-and-enable-cors) to enable CORS.
+* If the "There was an unexpected internal error. Please try again." message appears instead of the Sign-In Widget, then verify that CORS is enabled. Follow the steps in [Add a trusted origin and enable CORS](/docs/guides/oie-embedded-common-org-setup/go/main/#add-a-trusted-origin-and-enable-cors) to enable CORS.
+
+### Start your work with the use cases
+
+After you have successfully run the sample app, the next step is to build your integration by using the sample app as your guide. See [Load the Widget](/docs/guides/oie-embedded-widget-use-case-load/go/main/) to start using the Widget and explore the available use cases.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/go/runwidgetapp.md:32]:
```suggestion
After you successfully run the sample app, the next step is to build your integration by using the sample app as your guide. See [Load the Widget](/docs/guides/oie-embedded-widget-use-case-load/go/main/) to start using the Widget and explore the available use cases.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/index.md [hunk 7]
```diff
@@ -7,22 +7,25 @@ layout: Guides
 <ApiLifecycle access="ie" /><br>
 <ApiLifecycle access="Limited GA" /><br>
 
-This guide covers how you can run both the sample Widget app and the sample SDK app, after you have successfully [created and set up your Okta org](/docs/guides/oie-embedded-common-org-setup/) and [downloaded and set up the SDK](/docs/guides/oie-embedded-common-download-setup-app).
+<StackSelector />
+
+This guide covers how you can run the embedded Okta Identity Engine SDK or Sign-In Widget sample app, after you have successfully [created and set up your Okta org](/docs/guides/oie-embedded-common-org-setup/) and [downloaded and set up the SDK](/docs/guides/oie-embedded-common-download-setup-app).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/index.md:12]:
```suggestion
This guide covers how you can run the embedded Okta Identity Engine SDK or Sign-In Widget sample app after you successfully [create and set up your Okta org](/docs/guides/oie-embedded-common-org-setup/) and [download and set up the SDK](/docs/guides/oie-embedded-common-download-setup-app).
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/ios/runsdkapp.md [hunk 8]
```diff
@@ -7,3 +7,7 @@
 1. If you didn't didn't configure the `Okta.plist` file, you can add your org settings to the Configuration
    page by tapping the **Configure** link on the start-up screen. See [Configuration page](/docs/guides/oie-embedded-common-download-setup-app/ios/main/#configuration-page) for more information.
 1. Tap the **Sign In** button to start the authentication flow.
+
+### Work with the use cases
+
+After you've successfully run the sample app, you can build your own integration by using the sample app as your guide. Explore use cases that are available with the SDK, starting with the [Basic sign-in flow using the password factor](/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/ios/main/) use case.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/ios/runsdkapp.md:13]:
```suggestion
After you successfully run the sample app, you can build your own integration by using the sample app as your guide. Explore use cases that are available with the SDK, starting with the [Basic sign-in flow using the password factor](/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/ios/main/) use case.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/java/runsdkapp.md [hunk 9]
```diff
@@ -26,3 +26,7 @@
 1. Click **Login**. After you've successfully signed in, the app redirects you to a page that displays your basic user profile and security token information.
 
 1. Click **Logout** in the upper-right corner of the page to sign out of the sample app.
+
+### Work with the use cases
+
+After you've successfully run the sample app, you can build your own integration by using the sample app as your guide. Explore use cases that are available with the SDK, starting with the [Basic sign-in flow using the password factor](/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/java/main/) use case.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/java/runsdkapp.md:32]:
```suggestion
After you successfully run the sample app, you can build your own integration by using the sample app as your guide. Explore use cases that are available with the SDK, starting with the [Basic sign-in flow using the password factor](/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/java/main/) use case.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/java/runwidgetapp.md [hunk 10]
```diff
@@ -32,3 +32,7 @@
 1. Click **My Profile** in the upper-left corner of the page to view your basic user profile information.
 
 1. Click **Logout** in the upper-right corner of the page to sign out of the sample app.
+
+### Start your work with the use cases
+
+After you have successfully run the sample app, the next step is to build your integration by using the sample app as your guide. See [Load the Widget](/docs/guides/oie-embedded-widget-use-case-load/java/main/) to start using the Widget and explore the available use cases.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/java/runwidgetapp.md:38]:
```suggestion
After you successfully run the sample app, the next step is to build your integration by using the sample app as your guide. See [Load the Widget](/docs/guides/oie-embedded-widget-use-case-load/java/main/) to start using the Widget and explore the available use cases.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/nodejs/runsdkapp.md [hunk 11]
```diff
@@ -14,8 +14,12 @@
...
 
-* If the "There was an unexpected internal error. Please try again." message is displayed instead of the Sign-In Widget, then verify that CORS is enabled. Follow the steps in [Add a trusted origin and enable CORS](/docs/guides/oie-embedded-common-org-setup/nodejs/main/#add-a-trusted-origin-and-enable-cors) to enable CORS.
+* If the "There was an unexpected internal error. Please try again." message appears instead of the Sign-In Widget, then verify that CORS is enabled. Follow the steps in [Add a trusted origin and enable CORS](/docs/guides/oie-embedded-common-org-setup/nodejs/main/#add-a-trusted-origin-and-enable-cors) to enable CORS.
+
+### Work with the use cases
+
+After you've successfully run the sample app, you can build your own integration by using the sample app as your guide. Explore use cases that are available with the SDK, starting with the [Basic sign-in flow using the password factor](/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/nodejs/main/) use case.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/nodejs/runsdkapp.md:25]:
```suggestion
After you successfully run the sample app, you can build your own integration by using the sample app as your guide. Explore use cases that are available with the SDK, starting with the [Basic sign-in flow using the password factor](/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/nodejs/main/) use case.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/nodejs/runwidgetapp.md [hunk 12]
```diff
@@ -16,8 +16,12 @@ The sample app is located here: `okta-auth-js/samples/generated/express-embedded
...
 
-* If the "There was an unexpected internal error. Please try again." message is displayed instead of the Sign-In Widget, then verify that CORS is enabled. Follow the steps in [Add a trusted origin and enable CORS](/docs/guides/oie-embedded-common-org-setup/nodejs/main/#add-a-trusted-origin-and-enable-cors) to enable CORS.
+* If the "There was an unexpected internal error. Please try again." message appears instead of the Sign-In Widget, then verify that CORS is enabled. Follow the steps in [Add a trusted origin and enable CORS](/docs/guides/oie-embedded-common-org-setup/nodejs/main/#add-a-trusted-origin-and-enable-cors) to enable CORS.
+
+### Start your work with the use cases
+
+After you have successfully run the sample app, the next step is to build your integration by using the sample app as your guide. See [Load the Widget](/docs/guides/oie-embedded-widget-use-case-load/nodejs/main/) to start using the Widget and explore the available use cases.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/nodejs/runwidgetapp.md:27]:
```suggestion
After you successfully run the sample app, the next step is to build your integration by using the sample app as your guide. See [Load the Widget](/docs/guides/oie-embedded-widget-use-case-load/nodejs/main/) to start using the Widget and explore the available use cases.
```

**Review by susanharper-okta — APPROVED** [review]
couple of tense nits. Approving to not hold you up. Everything else looks great!

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/nodejs/runsdkapp.md [hunk 13]
```diff
@@ -14,8 +14,12 @@
 
    ![Example user profile page](/img/oie-embedded-sdk/Embedded-SIW-profile-page-nodejs.png)
 
-## Troubleshooting
+### Troubleshooting
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-run-samples/main/nodejs/runsdkapp.md:17]:
```suggestion
### Troubleshoot
```

**Pull request merged**